troposphere/firehose.py

```diff
@@ -216,7 +218,8 @@ class RedshiftDestinationConfiguration(AWSProperty):
         "ProcessingConfiguration": (ProcessingConfiguration, False),
         "RetryOptions": (RedshiftRetryOptions, False),
         "RoleARN": (str, True),
-        "S3Configuration": (S3Configuration, True),
+        "S3Configuration": (S3Configurclass S3DestinationConfiguration(AWSProperty):
```

Comment:
This change seems to have crept into the last commit. Looks like an editor typo, can you back this out? Thanks.